### ISSUE TITLE
Color schemes: Private badge text not readable on hover with Contrast color scheme

### ISSUE THREAD

**Issue by mattsherman** [Automattic/wp-calypso#36506]
When using the `Contrast` color scheme, the "Private" badge text in the site picker is not readable when hovering over a site in the sidebar list.

<img width="288" alt="ScreenCapture at Thu Oct 3 15:28:53 EDT 2019" src="https://user-images.githubusercontent.com/2098816/66158163-56590900-e5f3-11e9-8c4f-4a432ec02842.png">

From a quick test with a few other color schemes, this issue appears limited to the `Contrast` color scheme.
